src/infer_request.h

```diff
@@ -87,7 +88,8 @@ class InferRequest {
       const intptr_t request_address = 0,
       const PreferredMemory& preferred_memory =
           PreferredMemory(PreferredMemory::DEFAULT, 0),
-      const InferenceTrace& trace = InferenceTrace());
+      const InferenceTrace& trace = InferenceTrace(),
+      const uint32_t& request_release_flags = TRITONSERVER_REQUEST_RELEASE_ALL);
```

Comment:
Will a different flag be passed when creating a request object?

Reply:
Good catch! No, there won't be any. Fixed the constructor.